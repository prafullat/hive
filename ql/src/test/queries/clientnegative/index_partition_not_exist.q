CREATE INDEX srcpart_index_proj TYPE COMPACT ON TABLE srcpart(key);
ALTER INDEX srcpart_index_proj ON srcpart PARTITION(ds='2018-04-08', hr=12) REBUILD;
CREATE INDEX srcpart_index_proj TYPE SUMMARY ON TABLE srcpart(key) PARTITION(ds='2008-04-08', hr=11);
UPDATE INDEX srcpart_index_proj PARTITION(ds='2008-04-08', hr=12);