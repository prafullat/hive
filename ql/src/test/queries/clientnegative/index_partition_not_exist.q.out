PREHOOK: query: CREATE INDEX srcpart_index_proj TYPE COMPACT ON TABLE srcpart(key)
PREHOOK: type: CREATEINDEX
POSTHOOK: query: CREATE INDEX srcpart_index_proj TYPE COMPACT ON TABLE srcpart(key)
POSTHOOK: type: CREATEINDEX
FAILED: Error in semantic analysis: org.apache.hadoop.hive.ql.metadata.HiveException: Partition ds=2018-04-08/hr=12/ does not exist in table srcpart
PREHOOK: query: CREATE INDEX srcpart_index_proj TYPE COMPACT ON TABLE srcpart(key)
PREHOOK: type: CREATEINDEX
POSTHOOK: query: CREATE INDEX srcpart_index_proj TYPE COMPACT ON TABLE srcpart(key)
POSTHOOK: type: CREATEINDEX
FAILED: Error in semantic analysis: org.apache.hadoop.hive.ql.metadata.HiveException: Partition ds=2018-04-08/hr=12/ does not exist in table srcpart
PREHOOK: query: CREATE INDEX srcpart_index_proj TYPE COMPACT ON TABLE srcpart(key)
PREHOOK: type: CREATEINDEX
POSTHOOK: query: CREATE INDEX srcpart_index_proj TYPE COMPACT ON TABLE srcpart(key)
POSTHOOK: type: CREATEINDEX
FAILED: Error in semantic analysis: org.apache.hadoop.hive.ql.metadata.HiveException: Partition ds=2018-04-08/hr=12/ does not exist in table srcpart
PREHOOK: query: CREATE INDEX srcpart_index_proj TYPE COMPACT ON TABLE srcpart(key)
PREHOOK: type: CREATEINDEX
POSTHOOK: query: CREATE INDEX srcpart_index_proj TYPE COMPACT ON TABLE srcpart(key)
POSTHOOK: type: CREATEINDEX
FAILED: Error in semantic analysis: org.apache.hadoop.hive.ql.metadata.HiveException: Partition ds=2018-04-08/hr=12/ does not exist in table srcpart
